resources/constants.py

```diff
@@ -2145,6 +2145,250 @@ def lang_enum(name: str, values: Sequence[str]) -> Dict[str, str]:
     },
 }
 
+MOLD_PATTERNS = {
```

Comment:
General comment, what is the merit to having the mold patterns generated from data into models like this rather than using static block models?